build-system/tasks/make-extension/index.js

```diff
@@ -204,19 +205,44 @@ async function makeExtensionFromTemplates(
   const version = (
     options.version || (options.bento ? '1.0' : '0.1')
   ).toString();
+  const {'no_css': noCss} = options;
```

Comment:
In an effort to avoid negative boolean names, WDYT about `has_css` default to `true`? go/tott-530

Reply:
Unfortunately [minimist](https://www.npmjs.com/package/minimist) lacks a way to set arguments as `false`, it merely treats their presence as true. Hence all these flags prefixed `no`:

```
build-system/tasks/build.js:
  99:   noextensions: 'Builds with no extensions.',

build-system/tasks/default-task.js:
  76:   noconfig: 'Compiles production binaries without applying AMP_CONFIG',

build-system/tasks/dist.js:
  390:   noconfig: 'Compiles production binaries without applying AMP_CONFIG',
  395:   noextensions: 'Builds with no extensions.',

build-system/tasks/coverage-map/index.js:
  176:   nobuild: 'Skips dist build.',

build-system/tasks/runtime-test/helpers.js:
  86:     nobuild: 'Skipping build.',
```

Reply:
Ahh, thanks for the context!